.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,13 +2,13 @@ Thanks for submitting a PR, your contribution is really appreciated!
 
 Here's a quick checklist that should be present in PRs:
 
-- [ ] Target: for bug or doc fixes, target `master`; for new features, target `features`;
+- [ ] Add a new news fragment into the news folder
+  * name it $issue_id.$type
+  * if you dont have an issue_id change it to the pr id after creating the pr
+  * type is one of removal, feature, bugfix, vendor, doc, trivial
+- [ ] Target: for bbugfix, vendor, doc or trivial fixes, target `master`; for removals or features target `features`;
```

Comment:
(Sorry if I sound pedantic here, it is just that this will be seen for everyone contributing from now on so I think it pays to be precise and clear)